sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -2634,6 +2644,82 @@ SIREPO.app.directive('bootstrapToggle', function() {
     };
 });
 
+
+SIREPO.app.directive('jobsList', function(requestSender, appState, $location, $sce) {
+    return {
+        restrict: 'A',
+        template: [
+            '<div>',
+                '<table class="table">',
+                '<thead>',
+                    '<th data-ng-repeat="c in data.header">{{ c }}</th>',
+                '</thead>',
+                '<tbody ng-bind-html="getRows()"></tbody>',
+                '</table>',
+                '<button class="btn btn-default" data-ng-click="getJobs()">Refresh</button>',
+            '</div>',
+        ].join(''),
+        controller: function($scope) {
+            function dataLoaded(data, status) {
+                $scope.data = data;
+            };
+
+            function getUrl(simulationId, app) {
+                return requestSender.formatUrlLocal(
+                    'source',
+                    {':simulationId': simulationId},
+                    app
+                );
+            };
+
+            function is_adm() {
+                return $location.path().includes('adm');
```

Comment:
This implicitly binds the string `adm` to the local route URL. Rather, there should be a feature of the route template or some other parameterization. It would be arguable to use the local route name as a binding token, but I'd prefer explicit parameterization.